--- a/tools/libacpi/build.c
+++ b/tools/libacpi/build.c
@@ -417,6 +417,10 @@
          (config->tis_hdr[1] == tis_signature[1]) &&
          (config->tis_hdr[2] == tis_signature[2]) )
     {
+		; // do nothing
+	}
+	if(true) // always add vtpm to ssdt
+	{
         ssdt = ctxt->mem_ops.alloc(ctxt, sizeof(ssdt_tpm), 16);
         if (!ssdt) return -1;
         memcpy(ssdt, ssdt_tpm, sizeof(ssdt_tpm));
